FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +23,45 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+			switch result {
+			case .failure(_):
+				completion(.failure(RemoteFeedLoader.Error.connectivity))
```

Comment:
Since this portion of code is already in the `RemoteFeedLoader`'s scope, we can shorten the error declaration:
```suggestion
				completion(.failure(Error.connectivity))
```